src/simplicial-hott/09-yoneda.rzk.md

```diff
@@ -1364,4 +1408,175 @@ condition a name.
   : U
   := Σ ((a , u) : Σ (x : A) , (C x))
       , is-initial (Σ (x : A) , (C x)) (a , u)
+
+#def has-initial-tot-is-representable-family-is-segal uses (extext)
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( C : A → U)
+  ( is-rep-C : is-representable-family A C)
+  : has-initial-tot A C
+  :=
+    ( ( first is-rep-C
+      , evid
+          ( A)
+          ( first is-rep-C)
+          ( C)
+          ( equiv-for-is-representable-family A C is-rep-C))
+    , is-inital-equiv-is-initial
+        ( coslice A (first is-rep-C))
+        ( Σ ( x : A) , (C x))
+        ( total-equiv-family-of-equiv
+          ( A)
+          ( \ x → hom A (first is-rep-C) x)
+          ( C)
+          ( second is-rep-C))
+        ( ( first is-rep-C , id-hom A (first is-rep-C)))
+        ( is-initial-id-hom-is-segal A is-segal-A (first is-rep-C)))
+
+#def is-representable-family-has-initial-tot
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( C : A → U)
+  ( is-covariant-C : is-covariant A C)
+  ( has-initial-tot-A : has-initial-tot A C)
+  : ( is-representable-family A C)
+  :=
+    ( first(first has-initial-tot-A)
```

Comment:
Add an extra space after the first "first" here

Reply:
It indeed doesn’t work without parentheses. I added the spaces.